python/common/org/python/types/Float.java

```diff
@@ -450,7 +450,10 @@ public boolean __setattr_null(java.lang.String name, org.python.Object value) {
     )
     public org.python.Object __pow__(org.python.Object other, org.python.Object modulo) {
         if (modulo != null) {
-            throw new org.python.exceptions.NotImplementedError("float.__pow__() with modulo has not been implemented");
+            //throw new org.python.exceptions.NotImplementedError("float.__pow__() with modulo has not been implemented");
+            if (other instanceof org.python.types.Float) {
```

Comment:
Actually you don't need to perform this check, as `this` itself is a `Float` and using `modulo` should be disallowed regardless of whether `other` is an instance of `org.python.types.Float` or not.